src/turnkeyml/common/filesystem.py

```diff
@@ -365,6 +365,12 @@ class Keys:
     # Prefix for reporting the execution status of a stage
     # In the report this will look like stage_status:STAGE_NAME
     STAGE_STATUS = "stage_status"
+    # Parent key that holds all of the arguments to turnkey's
+    # evaluate_file() API
+    EVALUATION_ARGS = "args"
+    # Records the date and time of the evaluation after analysis but before
+    # build and benchmark
+    TIMESTAMP = "timestamp"
```

Comment:
Just curious, what prompted the addition of timestamp?

Reply:
We had a user who was running evaluations into a single cache over the course of a few months. Some combination of actions on their part broke turnkey. It would have been much simpler to diagnose if we had know the timing and order of those evaluations.